test-suite/handwritten-src/objc/tests/DBWcharTests.m

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+#import "DBWcharTestHelpers.h"
+#import "DBWcharTestRec.h"
+#import <XCTest/XCTest.h>
+
+@interface DBWcharTests : XCTestCase
+
+@end
+
+@implementation DBWcharTests
+
+- (void)setUp
+{
+    [super setUp];
+}
+
+- (void)tearDown
+{
+    [super tearDown];
+}
+
+- (void)test
+{
+    NSString *str1 = @"some string with unicode \u263a symbol";
+    NSString *str2 = @"another string with unicode \u263b symbol";
```

Comment:
Please test some non-BMP characters (e.g. U+1F4A9, encodable in ObjC as `\U0001F4A9`, or most other emoji) as well